principles/testing.md

```diff
@@ -127,6 +89,29 @@ Manual Quality Assurance activities will be performed on both UI and API for eac
 ### User Acceptance Testing:
   - UAT is out of scope for the product test team.
   - Model Office team will perform additional tests to verify the product fits the purpose from Service E2E point of view
+  
+## Testing Techniques
+  
+### Manual Testing
+  * Positive Testing
+  * Negative Testing
+  * Exploraroty Testing
+  
+### Test Automation
+
+![testing pyramid](./images/test-pyramid.png)
+
+  * Unit Tests
+  * Integration Tests
+  * API Tests
+  * UI Tests
+
+  **NOTE: Test automation code must follow the [Coding Principles](https://skillsfundingagency.github.io/das-technical-guidance/principles/coding-principles.html#coding-principles)**
```

Comment:
Here I'd add a [permalink](https://jekyllrb.com/docs/permalinks/) to the coding principles document and create a link using that reference (and I'd use a relative path rather than an absolute path like:

`See [Naming things]({{ 'standards/naming-things' | relative_url  }})`